### PR TITLE
[DellEMC] Dell Platform Modules Debian Build Error Fix

### PR DESCRIPTION
**Pull request by santhosh-kt**
**- What I did**
 - Fix for [Azure/sonic-buildimage#4095](https://github.com/Azure/sonic-buildimage/issues/4095)
- Exit status from failed make command(action) didnt reached parent target because the make command is inside the "for" loop.
- Only the exit status of the last command in the last iteration of the for loop is read by parent target.
- This is the reason why dpkg-buildpackage ignored the make error.
-  Fixed the issue with help of "set -e".

**- How I did it**
On branch s6000-build-error
Changes to be committed:
(use "git reset HEAD ..." to unstage)

        modified:   platform/broadcom/sonic-platform-modules-dell/s6000/modules/dell_s6000_platform.c
**- How to verify it**
Attached UT Logs:
Before Fix - [platform-modules-z9100_1.1_amd64.deb-without-fix.log.txt](https://github.com/Azure/sonic-buildimage/files/4153241/platform-modules-z9100_1.1_amd64.deb-without-fix.log.txt)

 
After Fix - [platform-modules-z9100_1.1_amd64.deb-with-fix.log.txt](https://github.com/Azure/sonic-buildimage/files/4153240/platform-modules-z9100_1.1_amd64.deb-with-fix.log.txt)
After Fix and platfrom debian completed with no issues log - [platform-modules-z9100_1.1_amd64.deb-no-error.log.txt](https://github.com/Azure/sonic-buildimage/files/4153441/platform-modules-z9100_1.1_amd64.deb-no-error.log.txt)

**- Description for the changelog**

[DellEMC] Dell Platform Modules Debian Build Error Fix

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by santhosh-kt** [comment]
Will check and update the build failure message.

**Comment by santhosh-kt** [comment]
Note: Upon build failure on vendor specific files , until the failure is addressed, further builds will be stopped.

File: platform/broadcom/sonic-platform-modules-dell/debian/rules [hunk 1]
```diff
@@ -37,7 +38,8 @@ override_dh_auto_build:
 		fi; \
 		echo "making man page alias $$mod -> $$mod APIs";\
 		make -C $(KERNEL_SRC)/build M=$(MOD_SRC_DIR)/$${mod}/modules; \
-	done)
+	done); \
+	set +e;
```
Comment (lguohan) [inline platform/broadcom/sonic-platform-modules-dell/debian/rules]: what about failure in the install phase?

**Comment by padmanarayana** [comment]
Fixes https://github.com/Azure/sonic-buildimage/issues/4095

File: platform/broadcom/sonic-platform-modules-dell/debian/rules (hunk 1 above)
Comment (santhosh-kt) [reply platform/broadcom/sonic-platform-modules-dell/debian/rules]: Updated.

**Pull request merged**